Description: Avoid CMPLX; different definitions under PGI and g++
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2019-01-15
Forwarded: no

Index: flang-20181226/runtime/libpgmath/lib/common/sincos/fd_sincos_scalar.cpp
===================================================================
--- flang-20181226.orig/runtime/libpgmath/lib/common/sincos/fd_sincos_scalar.cpp
+++ flang-20181226/runtime/libpgmath/lib/common/sincos/fd_sincos_scalar.cpp
@@ -179,7 +179,7 @@ __attribute__((noinline)) FCN_NAME(doubl
 
 
 #if     defined(SINCOS)
-#if     defined(CMPLX)
+#if     0
     return CMPLX(rs, rc);
 #else
     struct {
